airflow/plugins/utils.py

```diff
@@ -1,8 +1,80 @@
-import pandas as pd
+import abc
+import base64
+import gzip
+import json
+import logging
+import os
 import re
+import typing
+from enum import Enum
+from typing import ClassVar, List, Union
+from typing import Optional, Dict
 
+import gcsfs
+import humanize
+import pandas as pd
+import pendulum
+from airflow.models import Variable
 from calitp import read_gcfs, save_to_gcfs
+from calitp.config import is_development
+from calitp.storage import get_fs
 from pandas.errors import EmptyDataError
+from pydantic import AnyUrl, validator, Field
+from pydantic import BaseModel
+from pydantic.class_validators import root_validator
+from pydantic.tools import parse_obj_as
+from requests import Request
+from typing_extensions import Annotated, Literal
+
+
+def get_auth_secret(auth_secret_key: str) -> str:
+    try:
+        secret = Variable.get(auth_secret_key, os.environ[auth_secret_key])
+        return secret
+    except KeyError as e:
+        logging.error(f"No value found for {auth_secret_key}")
+        raise e
+
+
+def make_name_bq_safe(name: str):
```

Comment:
not sure why this is showing up in the diff here? 👀 assume it's ok but just odd

Reply:
that is odd... I will rebase to double checK?

Reply:
this is still weird... just want to triple check?

Reply:
ok I think this is fine and just a consequence of the rebase